manager/Makefile

```diff
@@ -46,6 +46,10 @@ run-integration-tests: export USE_MOCKUP_CONNECTOR?=true
 run-integration-tests: wait_for_manager
 	NO_SIMULATED_PROGRESS=true USE_EXISTING_CONTROLLER=true USE_EXISTING_CLUSTER=true go test ./... -v -run TestAPIs -count 1
 
+.PHONY: apply-tls-certificates
```

Comment:
you use it only once from `run-notebook-readflow-tls-tests`, why do you need a separate rule ?

Reply:
I can merge it with the rule in the root Makefile.